[WFLY-8741] credential-store operations intead of alias subresources - tests update

thaarok opened the pull request:
Tests update for [WFCORE-2777]
needs coordination with https://github.com/wildfly-security-incubator/wildfly-core/pull/158

mnovak committed:
WFLY-8751 test fix MDB20TopicTestCase testEjb20TopicMDBs - assert that subscriptions on topic were created before messages are sent to topic

Commit:
[WFLY-8624] Create test case for http-interface authentication secured by Elytron http-authentication-factory

fjuma committed:
[WFLY-8721] Fix test failures that occur when different values for node0 and node1 are passed to manual-mode, multi-node, and domain tests

rpelisse committed:
WFLY-8335 - User should be informed when switching between JDBC and journal store in transactions subsystem

Commit:
[WFLY-8741] credential-store operations intead of alias subresources - tests update

rhusar committed:
Code cleanup

rhusar committed:
WFLY-8752 Deprecate modcluster subsystem /ssl=configuration resource

psakar committed:
[WFLY-8776] Include wildfly-ejb-client-legacy-bom and jboss-ejb-client-legacy in maven repo

kabir committed:
Merge pull request #10026 from fjuma/WFLY-8721

[WFLY-8721] Fix test failures that occur when different values for node0 and node1 are used

jmesnil committed:
[WFLY-8746] RejectedExecutionException in the MDB during shutdown

The PooledConnectionFactoryService was is used by MDB was not depending
on the service controlling the lifecycle of Artemis client thread pools.
It was possible that the thread pools were cleaned (and stopped) before
the MDB was stopped causing RejectedExecutionException during MDB code
execution.

JIRA: https://issues.jboss.org/browse/WFLY-8746

pferraro committed:
WFLY-8594 Loading custom infinispan cache store from custom module fails

pferraro committed:
Replace redundant ComponentServiceNameFactory with ResourceServiceNameFactory.

pferraro committed:
ServiceNameProvider implementation for capabilities.

pferraro committed:
ServiceNameProvider implementation for capabilities.

pferraro committed:
Consolidate cache store capabilities.

pferraro committed:
Cleanup cache component/capability dependency logic.

pferraro committed:
Remove deployment unit attachment on undeploy or provide explanation for why it is retained

pferraro committed:
Generalize election policy capability.

pferraro committed:
Cosmetic

pferraro committed:
Cleanup capability dependency logic.

pferraro committed:
WFLY-8585 /subsystem=jgroups/channel=foo/fork=bar:remove operation leaks resources

kabir committed:
Merge pull request #10056 from pferraro/clustering

WFLY-8585 /subsystem=jgroups/channel=foo/fork=bar:remove operation leaks resources

kabir committed:
Merge pull request #10050 from rhusar/WFLY-8752

WFLY-8752 Deprecate modcluster subsystem /ssl=configuration resource

kabir committed:
Merge pull request #10052 from jmesnil/WFLY-8746_RejectedExecutionException

[WFLY-8746] RejectedExecutionException in the MDB during shutdown

kabir committed:
Merge pull request #10047 from mnovak1/master

WFLY-8751 test fix MDB20TopicTestCase testEjb20TopicMDBs - assert tha…

kabir committed:
Merge pull request #10051 from kwart/WFLY-8624-https-authentication-factory-tests

[WFLY-8624] Create test case for http-interface authentication secured by Elytron http-authentication-factory

kabir committed:
Merge pull request #10055 from pferraro/capability

WFLY-8594 Loading custom infinispan cache store from custom module fails

kabir reviewed (approved):
Approving paperwork. Also to me the implementation looks ok

pkremens committed:
[WFLY-8777] Allow skipTests in mixed-domain module

pferraro committed:
WFLY-8781 Default Capability#resolve(PathAddress) should utilize configured dynamic name mapper.

jmesnil committed:
[WFLY-8780] fix RejectedExecutionException when JMS bridge is reloaded

Add a dependency to the Artemis thread pool so that if either the source
or target JMS broker corresponds to a local Artemis server, the pool will
be cleaned up after the JMS bridge is stopped.

JIRA: https://issues.jboss.org/browse/WFLY-8780

kabir committed:
Merge pull request #10067 from jmesnil/WFLY-8780_jms-bridge_thread_pool

[WFLY-8780] fix RejectedExecutionException when JMS bridge is reloaded

kabir committed:
Merge pull request #10065 from pferraro/capability

WFLY-8781 Default Capability#resolve(PathAddress) should utilize configured dynamic name mapper.

kabir committed:
Merge pull request #10063 from pkremens/wfly-8777

[WFLY-8777] Allow skipTests in mixed-domain module

kabir committed:
Merge pull request #10061 from psakar/WFLY-8776

[WFLY-8776] Include wildfly-ejb-client-legacy-bom and jboss-ejb-client-legacy in maven repo

kabir committed:
Merge pull request #10039 from rpelisse/WFLY-8335-2

WFLY-8335 - User should be informed when switching between JDBC and journal store in transactions subsystem
